semgrep/semgrep/cli.py

```diff
@@ -159,9 +160,9 @@ def cli() -> None:
     config.add_argument(
         "--timeout",
         type=int,
-        default=0,
+        default=DEFAULT_TIMEOUT,
```

Comment:
Weird that you have to specify the default value here and in the main() function in semgrep_main.py
@nbrahms some Python tricks to avoid the duplication?